### ISSUE TITLE
When setting timeout, it keeps getting interpreted as a str

### ISSUE THREAD

**Issue by Sam-Hall** [tomcatmanager/tomcatmanager#84]
Every method of customising timeout that I've tried leads to similar errors...

```Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/urllib3/util/timeout.py", line 140, in _validate_timeout
    if value <= 0:
TypeError: '<=' not supported between instances of 'str' and 'int'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/cmd2/cmd2.py", line 1659, in onecmd_plus_hooks
    stop = self.onecmd(statement, add_to_history=add_to_history)
  File "/usr/local/lib/python3.6/site-packages/cmd2/cmd2.py", line 2062, in onecmd
    stop = func(statement)
  File "/usr/local/lib/python3.6/site-packages/tomcatmanager/interactive_tomcat_manager.py", line 735, in do_connect    r = self.tomcat.connect(url, user, password)
  File "/usr/local/lib/python3.6/site-packages/tomcatmanager/tomcat_manager.py", line 182, in connect
    r = self._get('serverinfo')
  File "/usr/local/lib/python3.6/site-packages/tomcatmanager/tomcat_manager.py", line 110, in _get
    timeout=self.timeout,
  File "/usr/local/lib/python3.6/site-packages/requests/api.py", line 76, in get
    return request('get', url, params=params, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/requests/api.py", line 61, in request
    return session.request(method=method, url=url, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/requests/sessions.py", line 530, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/local/lib/python3.6/site-packages/requests/sessions.py", line 643, in send
    r = adapter.send(request, **kwargs)
  File "/usr/local/lib/python3.6/site-packages/requests/adapters.py", line 435, in send
    timeout = TimeoutSauce(connect=timeout, read=timeout)
  File "/usr/local/lib/python3.6/site-packages/urllib3/util/timeout.py", line 96, in __init__
    self._connect = self._validate_timeout(connect, "connect")
  File "/usr/local/lib/python3.6/site-packages/urllib3/util/timeout.py", line 150, in _validate_timeout
    "int, float or None." % (name, value)
ValueError: Timeout value connect was 600, but it must be an int, float or None.
EXCEPTION of type 'ValueError' occurred with message: 'Timeout value connect was 600, but it must be an int, float or None.'
```

**Comment by kotfu** [comment]
Your stack trace is super helpful. There have been lots of changes in the upstream module ‘cmd2’. Upstream shipped 1.0.0 today, so things should stabilize. I can get this fixed quickly, maybe even tonight.

**Comment by Sam-Hall** [comment]
I tried working around it with this in tomcat_manager.py, line 110...
```
        r.response = requests.get(
            url,
            auth=(self.user, self._password),
            params=payload,
            timeout=int(self.timeout),
            )
```
but then I ran into issue #85 so then I went and hard coded a very long timeout for now
